ql/instruments/bond.cpp

```diff
@@ -205,12 +205,16 @@ namespace QuantLib {
                      Compounding comp,
                      Frequency freq,
                      Real accuracy,
-                     Size maxEvaluations) const {
+                     Size maxEvaluations,
+                     Real guess,
```

Comment:
You're taking a guess here, but you're not using it in the body as far as I can see?

Reply:
Ah, yes, oversight on my part. I intended to pass through both the guess and priceType. Will fix.